huaweicloud/services/acceptance/obs/resource_huaweicloud_obs_bucket_test.go

```diff
@@ -240,6 +240,30 @@ func TestAccObsBucket_logging(t *testing.T) {
 	})
 }
 
+func TestAccObsBucket_storageInfo(t *testing.T) {
+	rInt := acctest.RandInt()
+	resourceName := "huaweicloud_obs_bucket.bucket"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckOBS(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckObsBucketDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccObsBucket_basic(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckObsBucketExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "storage_info.0.size", "0"),
```

Comment:
if the config is same, pls check those two attr in Test**Basic

Reply:
done, this check has been removed to the basic test